finetuner-workflow/finetune-workflow.yaml

```diff
@@ -206,11 +272,25 @@ spec:
               --project_id {{workflow.parameters.project_id}}
               --epochs {{workflow.parameters.epochs}}
               --context_size {{workflow.parameters.context}}
+              --prompt_file {{=workflow.parameters.prompt_file == '' ? '' : '/' + workflow.parameters.pvc + '/' + workflow.parameters.prompt_file}}
```

Comment:
This is fixed in #195 by using `=` as an argument delimiter rather than a space, which correctly handles blank arguments (so just `--prompt-file=` works for specifying an empty string). Nice catch, thank you.